tests/framework/Optimizers/GradientDescent/gold/Basic/opt_export.xml

```diff
@@ -2,7 +2,7 @@
   <DataSet type="Static">
     <general>
       <inputs>trajID</inputs>
-      <outputs>x,y,func,const,ans,stepSize,iteration,accepted,conv_gradient</outputs>
+      <outputs>x,y,func,const,ans,aux_ans,stepSize,iteration,accepted,conv_gradient</outputs>
```

Comment:
Do you need this file? It seems to me this file can be removed.

Reply:
this file tests the output of the data from an optimization, and is already tested as part of the regression system (see `tests`.

Reply:
These variables are also included in the output csv file which are tested. It seems to me this file is not needed to be tested.